src/components/AdminForm.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+
+const API_BASE_URL = 'https://dev-kosj-api.fly.dev';
+
+const AdminForm: React.FC = () => {
+  const [email, setEmail] = useState('');
+  const [password, setPassword] = useState('');
+  const [isSuperAdmin, setIsSuperAdmin] = useState(false);
+
+  const handleSubmit = async (e: React.FormEvent<HTMLFormElement>) => {
+    e.preventDefault();
+    try {
+      // Assuming you have a function to get the token
+      const tokenResponse = await axios.post(`${API_BASE_URL}/login`, {
+        email: 'superadmin@gmail.com',
+        password: 'superadmin'
+      });
+      const token = tokenResponse.data.access_token;
+
+      const response = await axios.post(
+        `${API_BASE_URL}/accounts/`, 
+        { email, password, isSuperAdmin },
+        { headers: { Authorization: `Bearer ${token}` } }
+      );
```

Comment:
Dont directly call axios code, use a service as an intermediary as a way to clean your code and make things more organized